src/components/tabpanel/tabpanel-styles.jsx

```diff
@@ -1,7 +1,5 @@
 export default {
   root: {
-    padding: [20, 0],
```

Comment:
This is a breaking change but it's better to do this now than later

Reply:
ping @mjmonline , I think it's better if we add this padding in our component instead

Reply:
It can break visually, but the API of the component has not changed so it should be ok :)

Reply:
Looks good!